docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -134,70 +135,64 @@ Start by deleting the default files that come with the new project.
 
 1. To delete the old files:
 
-  ```sh
-  $ rm src/Add.ts
-  $ rm src/Add.test.ts
-  $ rm src/interact.ts
-  ```
+```sh
+$ rm src/Add.ts
+$ rm src/Add.test.ts
+$ rm src/interact.ts
+```
 
-1. Now, create the new files for your project:
+2. Now, create the new files for your project:
```

Comment:
the auto-numbering is in effect only when the spacing indents are retained

Reply:
@Pfed-prog  you inspired me to update the Docs Style Guide to add ordered list Markdown https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#numbered-steps - thank you!